ricecooker/classes/questions.py

```diff
@@ -159,6 +169,87 @@ def validate(self):
         except AssertionError as ae:
             raise InvalidQuestionException("Invalid question: {0}".format(self.__dict__))
 
+    def process_question(self, downloader):
+        """ process_question: Parse data that needs to have image strings processed
+            Args:
+                downloader (DownloadManager): download manager to download images
+            Returns: list of all downloaded files
+        """
+        image_files=[]
+        image_data = json.loads(self.raw_data)
+
+        # Process question
+        if 'question' in image_data and 'images' in image_data['question']:
+            image_data['question']['images'], qfiles = self.process_image_field(image_data['question'], downloader)
+            image_files += qfiles
+
+        # Process hints
+        if 'hints' in image_data:
+            for hint in image_data['hints']:
+                if 'images' in hint:
+                    hint['images'], hfiles = self.process_image_field(hint, downloader)
+                    image_files += hfiles
+
+        # Process answers
+        if 'answers' in image_data:
+            for answer in image_data['answers']:
+                if 'images' in answer:
+                    answer['images'], afiles = self.process_image_field(answer, downloader)
+                    image_files += afiles
+
+        # Process raw data
+        self.raw_data = json.dumps(image_data, ensure_ascii=False)
+        self.raw_data, data_files = super(PerseusQuestion, self).set_images(self.raw_data, downloader)
+
+        # Return all files
+        return image_files + data_files
+
+    def process_image_field(self, data, downloader):
```

Comment:
Why does `DownloadManager` need to be a separate parameter? How do you envision this to be used?